v2/tools/generator/internal/astmodel/type_name.go

```diff
@@ -227,6 +227,8 @@ const (
 	SpecSuffix = "_Spec"
 	// StatusSuffix is the suffix used for all Status types
 	StatusSuffix = "_STATUS"
+	// ArmSuffix is the suffix used for all ARM types
+	ArmSuffix = "_ARM"
```

Comment:
minor: I think `ARm` should be `ARM` here, with our styling. See for example `pipeline.CreateARMTypes` or `MakeSubscriptionScopeARMID`

Reply:
Fixed.